src/components/PageLayout/Navbar/NavbarMenu/Login/LoginSub.vue

```diff
@@ -21,12 +21,14 @@
 
     async login() {
       if (this.username && this.password) {
-        const data = await this.$store.dispatch('login', { username: this.username, password: this.password });
-        console.log(data);
+        const data = await this.$vxm.user.login({
+          username: this.username,
+          password: this.password,
+        });
         if (data.success) {
           this.$router.push('/labs/explore');
         } else {
-          this.$store.commit('showLoginFailedModal', { errorMessage: data.error });
+          this.$vxm.user.showLoginFailedModal({ errorMessage: data.error });
         }
       }
     }
```

Comment:
My bad on this, but this component shouldn't have anything inside of it.
I don't mind to remove it in a later commit, but if you're already editing it, you can drop it.

Reply:
Done